systemtests/init_test.go

```diff
@@ -204,7 +226,7 @@ func (s *systemtestSuite) SetUpTest(c *C) {
 
 		for _, node := range s.nodes {
 			//node.cleanupContainers()
-			node.cleanupDockerNetwork()
+			//node.cleanupDockerNetwork()
```

Comment:
From what i saw cleanupslave does the job. I have left it commented out for aci case - if needed. we can address it .